arangod/Aql/AqlFunctionsInternalCache.h

```diff
@@ -79,10 +84,12 @@ class RegexCache final {
 
  private:
   /// @brief cache for compiled regexes (REGEX function)
-  std::unordered_map<std::string, std::unique_ptr<icu::RegexMatcher>> _regexCache;
+  std::unordered_map<std::string, std::unique_ptr<icu::RegexMatcher>> _aqlFunctionsInternalCache;
```

Comment:
I don't think this renaming is correct. This map _is_ still a regex cache, so it should remain as `_regexCache`.